ert_storage/endpoints/records.py

```diff
@@ -1,7 +1,22 @@
 import json
```

Comment:
Just a general comment to this file, some unused imports which could be removed. Also pydantic is used but not listed as requirement in setup.py.